jvm/src/main/scala-2.10/scalajson.ast/unsafe/JValue.scala

```diff
@@ -174,6 +174,11 @@ final case class JObject(value: Array[JField] = Array.empty) extends JValue {
 
   override def hashCode: Int =
     java.util.Arrays.deepHashCode(value.asInstanceOf[Array[AnyRef]])
+
+  override def toString =
+    productIterator
+      .map(scala.runtime.ScalaRunTime.stringOf)
```

Comment:
Is having code that depends on `ScalaRunTime` for a library that is designed to be ultra stable considered wise?

The documentation for `ScalaRunTime` states

> The object ScalaRunTime provides support methods required by
 the scala runtime.  All these methods should be considered
 outside the API and subject to change or removal without notice.

Reply:
No, it's not. What alternative do you think scalajson should use?

Reply:
Its not the point about alternative, its the fact that (if I understand correctly), `ScalaRunTime` can change at any time, independent of the stability guarantees for binary compatibility for Scala.

Note that I don't know if this is the actual case, I am just going from what is stated in the documentation linked earlier

Reply:
Sorry, I think I wasn't clear in my last message. I agree with your concerns around the stability of `ScalaRunTime`'s API, as stated in its scaladoc.

I was wondering what alternatives should be pursued in its place.

Reply:
I will ask around to see if there is an alternative or whether this specific usage of `ScalaRunTime` is a free pass

Reply:
@dwijnand So I just spoke with @sjrd, and he said that using `ScalaRunTime` in any stable library is a very bad idea and it has caused him issues with Scala.js, i.e. see https://github.com/scala/scala/pull/5098

Would it be possible to just manually inline the code that is needed, its annoying but I think that since we are only working with Arrays/Object, it shouldn't be too much code.